static/src/stylesheets/module/atoms/temporary/_explainers.scss

```diff
@@ -1,158 +1,9 @@
-// This will will be removed when the atom library is handling
-// the rendering of all snippet atoms
+//style support for legacy explainer atoms
```

Comment:
Can you move that file out of the `temporary` please?

Reply:
Okay, can do